### PR TITLE
Demo test to show usage of Monitoring APIs

### PR DESCRIPTION
**Pull request by chandrams**
Demo test to show usage of Monitoring APIs

### CODE REVIEW

File: monitoring_demo/kruize/autotune.py [hunk 1]
```diff
@@ -0,0 +1,114 @@
+"""
```
Comment (dinogun) [inline monitoring_demo/kruize/autotune.py:1]: Please name this file as kruize.py, we dont want to highlight autotune here

File: monitoring_demo/kruize/autotune.py [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+from . json_validate import validate_exp_input_json
+import subprocess
+import requests
+import json
+import os
+import time
+
+def form_autotune_url(cluster_type):
```
Comment (dinogun) [inline monitoring_demo/kruize/autotune.py:24]: All autotune references here should be changed to indicate kruize

File: monitoring_demo/kruize/autotune.py (hunk 1 above)
Comment (chandrams) [reply monitoring_demo/kruize/autotune.py]: Done

**Review by dinogun — APPROVED** [review]
LGTM

**Pull request merged**